website/docusaurus.config.js

```diff
@@ -61,6 +61,20 @@ module.exports = {
         },
       ],
     },
+    metadata: [
+      {
+        name: 'msapplication-TileColor',
+        content: '#2b5797',
+      },
+      {
+        name: 'msapplication-config',
+        content: '/ts-node/img/favicon/browserconfig.xml',
+      },
+      {
+        name: 'theme-color',
+        content: '#ffffff',
+      },
+    ],
```

Comment:
This looks new, what does it do?  Do we need to make these colors match something else in our config?